openwisp_ipam/api/__init__.py

```diff
@@ -34,4 +34,5 @@ def get_api_urls(api_views):
         path('subnet/<str:pk>/', api_views.subnet, name='subnet'),
         path('subnet/<str:subnet_id>/hosts/', api_views.subnet_hosts, name='hosts',),
         path('ip-address/<str:pk>/', api_views.ip_address, name='ip_address'),
+        re_path('', include('openwisp_utils.api.urls')),
```

Comment:
You don't need to add these changes here. Please make these changes in [tests/openwisp2/urls.py](https://github.com/openwisp/openwisp-ipam/blob/master/tests/openwisp2/urls.py)

Also change it to following:
```
    path('^api/v1/', include('openwisp_utils.api.urls')),
```
These requested changes will ensure that we maintain consistency with other repositories of OpenWISP. You can look to these repositories for reference: 
- [openwisp-monitoring](https://github.com/openwisp/openwisp-monitoring/blob/master/tests/openwisp2/urls.py#L17)
- [openwisp-notifications](https://github.com/openwisp/openwisp-notifications/blob/master/tests/openwisp2/urls.py#L10)
- [openwisp-utils](https://github.com/openwisp/openwisp-utils/blob/master/tests/urls.py#L7)
- [openwisp-users](https://github.com/openwisp/openwisp-users/blob/master/tests/openwisp2/urls.py#L21)